website/docs/api/options-stack.mdx

```diff
@@ -114,6 +114,10 @@ Hide the TopBar when a scrolling layout is scrolled.
 | ------- | -------- | -------- |
 | boolean | No       | Both     |
 
+:::warning Deprecation warning
+This option will be deprecated in a future release. Please use the [OptionsSearchBar](topBar-searchBar.mdx) option instead.
```

Comment:
Shouldnt it be `removed` instead of `deprecated`? Afaik its deprecated now and will be removed in a future release. This applies to the following doc changes as well

Reply:
Maybe we should define it as `moved`? we can also point to the correct path `topBar.searchBar.*`.

Reply:
@yogevbd @danilobuerger makes sense. how the warning should look like?

Reply:
@mateioprea I've suggested a few changes above in this matter - the option is currently deprecated and will be removed in a future major version. If you accept my suggestions we can merge the PR :D

Reply:
@guyca cannot accept those once I've commited after you've left the comments. I just pushed a new commit. 💯